src/quo2/components/dividers/divider_label.cljs

```diff
@@ -25,12 +25,13 @@
     [rn/view
      {:accessible          true
       :accessibility-label :divider-label
-      :style               {:border-top-width   1
-                            :border-top-color   border-and-counter-bg-color
-                            :padding-top        padding-top
-                            :padding-horizontal 16
-                            :align-items        :center
-                            :flex-direction     :row}}
+      :style               (merge {:border-top-width   1
```

Comment:
consider refactoring this with best practices, i.e putting this into a `divider-label` folder and using a view and style file

Reply:
That's not part of the PR. But I think all quo2 components need refactoring.

Reply:
yep, they do :)